modules/ROOT/pages/to-find-info.adoc

```diff
@@ -154,19 +224,20 @@ the reason is provided.
 |Optimized API |optimized-api |sales:/connect |Yes
 |===
 
-== Search by Tags and Categories in Exchange Cloud
 
-To search by tags and categories in Exchange Cloud, use the drop-down menus for tags and categories on the search page.
+== Search by Types, Tags, Categories, and Custom Fields in the Original Exchange Search Experience
```

Comment:
Isn't this rather far removed from its equivalent for the new search, Line 24 above?

Reply:
Yes, after discussion with PM and devs we agreed to put sections exclusive to the old search at the bottom.